components/why-kedro/why-kedro.module.scss

```diff
@@ -1,10 +1,25 @@
 @import '../../styles/variables';
 @import '../../styles/mixins';
 
-.container {
+.outer {
+  background-color: $color-bg-grey;
+  padding: 5rem 0;
+
+  @include breakpoint(medium-up) {
+    margin: 0 -3.5rem;
+    padding: 5rem 3.5rem;
+  }
+
+  @include breakpoint(small) {
+    padding: 2.5rem 1rem 1.5rem;
```

Comment:
And here, too. More padding along the bottom.

Reply:
Also, we want the navigation to go off the screen on mobile, not have a space on the right side.

![image](https://user-images.githubusercontent.com/16869061/165917921-2a69cacc-1344-4f8b-a6e2-1ae0b1aae96c.png)

So we might have to give padding to each of the individual elements on mobile instead of to the container.